### PR TITLE
MM-1932 Added emoji wrapper with cross platform available gliffs

### PR DESCRIPTION
**Pull request by nickago**
This will wrap all the iOS emojis in a format that all computers can read, confirmed working on Windows Chrome and IE10. Also removed an unnecessary update force from the rhs which made the emojis reparse on intervals.

### CODE REVIEW

File: web/react/components/rhs_thread.jsx [hunk 1]
```diff
@@ -92,14 +89,6 @@ export default class RhsThread extends React.Component {
...
         }
     }
-    onTimeChange() {
-        for (var id in this.state.postList.posts) {
-            if (!this.refs[id]) {
-                continue;
-            }
-            this.refs[id].forceUpdate();
-        }
-    }
```
Comment (jwilander) [inline web/react/components/rhs_thread.jsx]: This isn't required any more to update the timestamps?

Comment (nickago) [inline web/react/components/rhs_thread.jsx]: yah, the timestamps are static on the right hand side

Comment (jwilander) [inline web/react/components/rhs_thread.jsx]: Ok, cool

File: web/react/components/rhs_comment.jsx [hunk 2]
```diff
@@ -12,12 +12,16 @@ var FileAttachmentList = require('./file_attachment_list.jsx');
...
+var twemoji = require('twemoji');
 
 export default class RhsComment extends React.Component {
     constructor(props) {
         super(props);
 
         this.retryComment = this.retryComment.bind(this);
+        this.parseEmojis = this.parseEmojis.bind(this);
+        this.componentDidMount = this.componentDidMount.bind(this);
+        this.componentDidUpdate = this.componentDidUpdate.bind(this);
```
Comment (jwilander) [inline web/react/components/rhs_comment.jsx]: For native React functions you don't need to bind this. So componentDidMount and componentDidUpdate are unnecessary here.

File: web/react/components/rhs_root_post.jsx [hunk 3]
```diff
@@ -6,12 +6,25 @@ var UserProfile = require('./user_profile.jsx');
...
+var twemoji = require('twemoji');
+var Constants = require('../utils/constants.jsx');
 
 export default class RhsRootPost extends React.Component {
     constructor(props) {
         super(props);
+
+        this.parseEmojis = this.parseEmojis.bind(this);
+        this.componentDidMount = this.componentDidMount.bind(this);
+        this.componentDidUpdate = this.componentDidUpdate.bind(this);
```
Comment (jwilander) [inline web/react/components/rhs_root_post.jsx]: For native React functions you don't need to bind this. So componentDidMount and componentDidUpdate are unnecessary here.

**Comment by crspeller** [comment]
Not ready to merge before refactoring work. Closing because will need rebase.

**Pull request closed without merging**